### PR TITLE
HoC 2023 - Code.org - Update Minecraft landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update https://code.org/minecraft 
- Page is fully responsive
- Works with LTR and RTL languages
- Deprecated unused legacy views

**Jira ticket:** [ACQ-958](https://codedotorg.atlassian.net/browse/ACQ-958)

----

localhost code org_3000_minecraft

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/minecraft/minecraft_education_activities.haml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      title: hoc_s(:minecraft_education_activity_title_2020),
+      image: "/images/mc/minecraft-activity-gen-ai-for-earth-2020.png",
+      desc: hoc_s(:minecraft_education_activity_desc_2020),
+      url: "https://education.minecraft.net/en-us/lessons/minecraft-hour-of-code",
+      aria_label: hoc_s(:aria_label_call_to_action_minecraft_2020)
+    },
+  ]
+
+-# Hiding this until the activity is ready
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/views/minecraft/minecraft_education_activities.haml:26]: Will set this live the week of 11/13 in a separate PR.

File: pegasus/sites.v3/code.org/public/css/page/minecraft-styles.scss [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.carousel {
+  margin: 1rem 0 2rem;
+
+  @media screen and (max-width: $width-sm) {
+    margin: 0 0 2rem;
+  }
+
+  &.blocks {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/minecraft-styles.scss:105]: Styles for the activity carousel.

File: pegasus/sites.v3/code.org/public/css/page/minecraft-styles.scss [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+      position: relative;
+
+      .slides {
+        width: 300%;
+        display: flex;
+      }
+    }
+  }
+
+  &.videos {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/minecraft-styles.scss:142]: Styles for the videos carousel.

File: pegasus/sites.v3/code.org/public/css/page/dance-styles.scss [hunk 4]
```diff
@@ -31,8 +31,8 @@
       height: 100%;
       background-image: url('/images/banners/banner-bg-maze-squares-light.png');
       background-repeat: repeat;
-      background-size: 30%;
-      z-index: 2;
+      background-size: contain;
+      z-index: 1;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/dance-styles.scss:35]: Making the code.org/dance page hero banner pattern size match the Minecraft page.

File: pegasus/sites.v3/code.org/public/js/page/minecraft_page.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    slidesContainer.appendChild(slideWrapper);
+  });
+
+  slideItems.forEach((slideItem) => slideItem.remove());
+}
+
+function handleScreenSize(carouselId, cardsList) {
+  if (windowWidth < 639) {
+    reorganizeAndWrapSlides(carouselId, 1);
+    console.log("> 639");
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/js/page/minecraft_page.js]: I think this should be "< 639" if I'm reading that right?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 6]
```diff
@@ -705,6 +725,28 @@
...
+  minecraft_education_activity_title_2022: "Minecraft Escape Estate"
+  minecraft_education_activity_desc_2022: "Escape the mysterious mansion by dawn and earn a million emeralds!"
+  minecraft_education_activity_title_2021: "Minecraft TimeCraft"
+  minecraft_education_activity_desc_2021: "Travel back in time to save the future in this free coding lesson in Minecraft Education."
+  minecraft_education_activity_title_2020: "Minecraft AI for Earth"
+  minecraft_education_activity_desc_2020: "Program the Minecraft Agent to collect data about forest fires. Learn coding basics and explore a real-world example of artificial intelligence."
+  minecraft_resources_desc_01: "Utilize this on demand [free course](%{course_url}) or register your entire class to attend a live Hour of Code workshop with a guest teacher. Offered at multiple times during CSEdWeek, %{campaign_date}."
+  minecraft_resources_desc_02: "Inspire STEM passion in K-12 learners with 200hr of computer science curriculum, from block to python, supported by professional development."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Should this say "200 hours" or is "200hr" okay?

**Review by TurnerRiley — APPROVED** [review]
Looks great! Just had a couple tiny questions

File: pegasus/sites.v3/code.org/public/js/page/minecraft_page.js (hunk 5 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/js/page/minecraft_page.js]: You're right! Updated.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 6 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Oh yes, that sounds much better. Updated!

**Pull request merged**